packages/react-scripts/config/webpack.config.js

```diff
@@ -523,13 +492,21 @@ module.exports = function(webpackEnv) {
       ],
     },
     plugins: [
+      new ExtensionReloader({
+        entries: {
+          contentScript: ['app', 'contentScript'],
+          background: 'background',
+        },
+      }),
```

Comment:
This causes the extension to reload when any part of the application's entry points are changed. There's more information on the configuration options here https://github.com/rubenspgcavalcante/webpack-extension-reloader